src/codegen/function.rs

```diff
@@ -456,3 +459,41 @@ pub fn body_chunk_futures(
 
     Ok(body)
 }
+
+pub(super) fn generate_static_to_str(
+    w: &mut dyn Write,
+    env: &Env,
+    function: &analysis::functions::Info,
+    scope_version: Option<Version>,
+) -> Result<()> {
+    writeln!(w)?;
+    let version = Version::if_stricter_than(function.version, scope_version);
+    version_condition(w, env, version, false, 1)?;
+
+    let visibility = match function.visibility {
+        Visibility::Public => "pub ",
+        _ => "",
+    };
+
+    writeln!(
+        w,
+        "\
+\t{visibility}fn {rust_fn_name}(self) -> &'static str {{
+\t\tunsafe {{
+\t\t\tCStr::from_ptr(
+\t\t\t\t{ns}::{glib_fn_name}(self.to_glib())
+\t\t\t\t\t.as_ref()
+\t\t\t\t\t.expect(\"{glib_fn_name} returned NULL\"),
+\t\t\t)
+\t\t\t.to_str()
+\t\t\t.unwrap()
```

Comment:
Please never use `unwrap()`, always `expect()`.

Reply:
Here I've adopted what [was already there](https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/blob/8b9ef8b10961ba2a9dad933c79219423253247ef/gstreamer-audio/src/audio_format.rs#L127-129) in manual code, will update both.

Reply:
Yes please! :)

Reply:
Done :)